src/compile/scale/range.ts

```diff
@@ -229,6 +264,46 @@ export function defaultRange(
   throw new Error(`Scale range undefined for channel ${channel}`);
 }
 
+export function defaultContinuousToDiscreteCount(
+  scaleType: ScaleType,
+  config: Config,
+  domain: Domain,
+  channel: Channel
+) {
+  switch (scaleType) {
+    case 'quantile':
+      return config.scale.quantileCount;
+    case 'quantize':
+      return config.scale.quantizeCount;
+    case 'threshold':
+      if (domain !== undefined && isArray(domain)) {
+        return domain.length + 1;
+      } else {
+        log.warn(log.message.domainRequiredForThresholdScale(channel));
+        // default threshold boundaries for threshold scale since domain has cardinality of 2
+        return 3;
+      }
+    default:
```

Comment:
I think we don't need default here since there are only 3  continuousToDiscrete scale types. (So this case is impossible.) 

Better modify the type `scaleType: ScaleType` to include only these 3 types and remove this case.

Reply:
Done